app/routers/event.py

```diff
@@ -98,6 +98,25 @@ async def eventview(request: Request, event_id: int,
                                        "messages": messages})
 
 
+@router.post("/{event_id}")
+async def changeowner(request: Request, event_id: int,
+                      db: Session = Depends(get_db)):
+    data = await request.form()
+    if 'username' in data:
```

Comment:
Prefer `if 'username' not in form: return [...]` and the rest of the code unindented